Dockerfile

```diff
@@ -59,13 +104,26 @@ RUN apt-get -y install \
 #
 # swig: is required for different purposes
 # https://github.com/rtfd/readthedocs-docker-images/issues/15
-RUN apt-get -y install plantuml imagemagick librsvg2-bin swig
+RUN apt-get -y install \
+      imagemagick \
+      librsvg2-bin \
+      plantuml \
+      swig
 
 # Install Python tools/libs
-RUN apt-get -y install python-pip && pip install -U virtualenv auxlib
+RUN apt-get -y install \
+      python-pip \
+ && pip install -U \
```

Comment:
Maybe it's better to continue with the visual indentation here.

Reply:
Sorry, but I don't get this. Do you want me to write it this way?!?

```
RUN apt-get -y install \
      python-pip \
    && pip install -U \
```

Reply:
```
RUN apt-get -y install \
      python-pip \
   && pip install -U \
```

or 

```
RUN apt-get -y install \
      python-pip \
      && pip install -U \
```

or

```
RUN apt-get -y install \
      python-pip && \
      pip install -U \
```

I think 1) and 3) are more readable. 1) looks nicer to me ;)

Reply:
All in all, you're aligning commands to package arguments. Really?

In the current code, I've aligned the `pip` to the